web-registry/src/pages/ProjectCreate/ProjectCreate.context.tsx

```diff
@@ -0,0 +1,32 @@
+import { createContext, ReactNode, useContext, useState } from 'react';
```

Comment:
moved it to this new route because I thought it made more sene there - tbh I'm not sure I fully understand how we plan to use the `/ecocredit` module, so feel free to move back if you think it makes more sense there